libs/angular/src/lib/dropdown/dropdown.component.ts

```diff
@@ -299,13 +298,16 @@ export class NggDropdownComponent
   }
 
   private setSelectionByValue(value: any) {
-    if (this._value !== value) {
-      this._value = value
-
-      const selected = this.handler?.dropdown?.options.find(
-        (option) => option.value === value
-      )
-      if (selected) this.handler?.select(selected)
+    if (!this.dropdown?.isMultiSelect) {
```

Comment:
Should the multiselect also be able to set selection by value input?

Reply:
@Maximilian-R that was my intention to begin with but it was a bit tricky to implement with the current setup so decided to skip that use case for now as it's possible to work around it by using the selected property:)